### ISSUE TITLE
CPU Load

### ISSUE THREAD

**Issue by Rodov** [5e/streamdeck-hwinfo-plugin#32]
CPU load more than HWiNFO plugin. Can you fix it?

**Comment by 5e** [comment]
I'm not sure what you mean. Do you mean this? https://github.com/5e/streamdeck-hwinfo-plugin/issues/30

**Comment by Rodov** [comment]
No. I mean HWiNFO plugin load CPU no more than 10%, while HWiNFO Reader loads CPU more than 10% with the same sensors displaying.

**Comment by 5e** [comment]
Im not sure why it would show different values, all the plugin does is read the values from HWiNFO. It could be a problem on their end. 

Some screenshots would be useful in debugging this

**Comment by Rodov** [comment]
The same sensor. This happening not every time, but sometimes.
The screenshot bellow with two programs at the same time:
![screenshot](https://github.com/user-attachments/assets/b06d122d-c534-4abe-aabd-91222b9ccaec)

Screenshots with programs setup:
![Screenshot 2024-11-02 173620](https://github.com/user-attachments/assets/caeb23f1-a149-488a-b1b0-93f90a23e197)
![Screenshot 2024-11-02 173649](https://github.com/user-attachments/assets/23b12691-458e-4898-902e-43059d5d0413)

**Comment by 5e** [comment]
Interesting, I'm not sure what would cause this. I'm just relaying information from HWiNFO to the screen. I'll have a look into it.

**Comment by Rodov** [comment]
Thanks a lot !

**Comment by VinzNL** [comment]
Isn't this just a latency thing? I just switched over from the old plug-in to this plug-in and noticed it takes a bit longer to report values, so they're out of sync with the old plug-in.
